Source/Core/CesiumTerrainProvider.js

```diff
@@ -633,9 +652,10 @@ import TileProviderError from './TileProviderError.js';
             return undefined;
         }
 
-        var yTiles = provider._tilingScheme.getNumberOfYTilesAtLevel(level);
-
-        var tmsY = (yTiles - y - 1);
+        if (provider._tilingScheme instanceof GeographicTilingScheme) {
```

Comment:
Quantized mesh tile URL Y coordinates should always be numbered from the South and increasing toward the North, TMS-style. It's confusing for this to change based on the projection.

Reply:
While we're at it, we can make this configurable separately right? Maybe when the `scheme` key is set to `tms` it counts from the south, and counts from the north otherwise. Or to avoid breaking existing uses, a new `slippy` value for scheme triggers counting from the north.

Reply:
That's ok with me if you have a use case and as long as it doesn't add too much complexity to the URL generation (it shouldn't be too bad).